creational/src/main/java/com/patterns/builder/Supermarket.java

```diff
@@ -1,4 +1,4 @@
-package org.patterns.creational;
+package com.patterns.builder;
 
 public class Supermarket {
```

Comment:
`Supermarket.Builder` must also implement `build` to create Supermarket from builder object.

Reply:
e0355826eb9d6d4f2b55bad432d2b48364e15ec8